Tests/PointFreeTests/BlogTests.swift

```diff
@@ -53,7 +53,7 @@ class BlogTests: TestCase {
   }
 
   func testBlogShow() {
-    let req = request(to: .blog(.show(.right(1))), basicAuth: true)
+    let req = request(to: .blog(.show(.right(post0000_mock.id.unwrap))), basicAuth: true)
```

Comment:
Should these take proper ids?

Reply:
I have a checklist item to revamp how we link to blog posts.